packages/oui-guide-dropdown/README.md

```diff
@@ -0,0 +1,134 @@
+# Guide Dropdown
+
+<component-status cx-design="complete" ux="complete"></component-status>
+
+## Usage
+
+
+```html:preview
+<oui-guide title="Guide name">
+  <oui-guide-body>
+    <oui-action-menu-item text="External link" href="#" external></oui-action-menu-item>
+    <oui-action-menu-item text="External link" href="#" external></oui-action-menu-item>
+    <oui-action-menu-item text="External link" href="#" external></oui-action-menu-item>
+  </oui-guide-body>  
+</oui-guide>
+```
+
+## Without title
+
+
+```html:preview
+<oui-guide title="">
+  <oui-guide-body>
+    <oui-action-menu-item text="External link" href="#" external></oui-action-menu-item>
+    <oui-action-menu-item text="External link" href="#" external></oui-action-menu-item>
+    <oui-action-menu-item text="External link" href="#" external></oui-action-menu-item>
+  </oui-guide-body>  
+</oui-guide>
+```
+
+## With sections and long texts
+
+
+```html:preview
+<oui-guide title="Guides">
+  <oui-guide-body>  
+    <oui-action-menu-item text="External link" href="#" external></oui-action-menu-item>
+    <oui-guide-section title="Section1">
+      <oui-action-menu-item text="External link" href="#" external></oui-action-menu-item>
+      <oui-action-menu-item text="External link" href="#" external></oui-action-menu-item>
+    </oui-guide-section>
+    <oui-guide-section title="Section2 verrrrrrrrrrrrrrrrrrrrrrrrrrrry very long title for section title">

```

Comment:
Maybe "very very very" or another long text instead of /ve(r+)y/ ^^

Reply:
It's not the same test though ! `very very very` won't be handled the same way as `verrrrrrrrrrrrrrrrrrrrrry`

Reply:
I think it's more like for testing how the component handle the wordbreaking :)

Reply:
Well I understand. But I don't like it because all our weird test cases are exposed in the documentation website. I think we'll have to make our own development pages in the future.